Snippets/Snippets_5/Features/FeatureConfiguration.cs

```diff
@@ -0,0 +1,79 @@
+﻿namespace Snippets5.Features
+{
+    using System;
+    using NServiceBus;
+    using NServiceBus.Config;
+    using NServiceBus.Features;
+    using NServiceBus.Pipeline;
+
+    class SecondLevelRetries : Feature
+    {
+
+#region FeatureConfiguration
+        public SecondLevelRetries()
+        {
+            EnableByDefault();
+
+            DependsOn<DelayedDeliveryFeature>();
+
+            Prerequisite(context => !context.Settings.GetOrDefault<bool>("Endpoint.SendOnly"), "Send only endpoints can't use SLR since it requires receive capabilities");
```

Comment:
Nitpicking here, but would error message like this be clearer? "SecondLevelRetries requires endpoint with receive capabilities and cannot be used with send-only endpoint."